src/useKeyboard.ts

```diff
@@ -50,17 +50,17 @@ export function useKeyboard() {
     ]
 
     return () => {
-      if (Keyboard.removeListener) {
```

Comment:
This check work until the RN team decided to restore this method: https://github.com/facebook/react-native/commit/035718ba97bb44c68f2a4ccdd95e537e3d28690c

Reply:
My recommendation -- do a version check >= 0.65 -> use `.remove`. and < 0.65 -> use `.removeListener` to support all versions of react native

Reply:
I checked  0.60.x , 0.61.x, 0.62.x  RN versions

and in these versions addEventListner returned subscription with remove method